libportal/account.c

```diff
@@ -23,9 +23,8 @@
 #include "portal-private.h"
 
 /**
- * SECTION:account
- * @title: Accounts
- * @short_description: basic user information
+ * Accounts
```

Comment:
This won't be rendered.

You should replace the `/**` at the top with `/*`, as this block is not a documentation annotation.